### PR TITLE
Fix stuff

### PR DESCRIPTION
**Pull request by lylythechosenone**
Fixed some various stuff (mostly beatifying) and realized that [this issue](https://github.com/ANF-Studios/WinPath/issues/32) wasn't an issue at all.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -365,3 +365,6 @@ MigrationBackup/
 
 # Fody - auto-generated XML schema
 FodyWeavers.xsd
+
+.vscode
```
Comment (ANF-Studios) [inline .gitignore:369]: I prefer not to ignore `.vscode`

File: WinPath/src/Update.cs [hunk 2]
```diff
@@ -5,23 +5,24 @@
...
 namespace WinPath
 {
     public class Update
     {
         private readonly bool includePrereleases;
-        public readonly bool Is32Or64BitOperatingSystem;
+        public readonly bool is32Or64BitOperatingSystem;
         private bool confirmDownload;
-        private const string releases = "https://api.github.com/repos/ANF-Studios/WinPath/releases";
+        private const string Releases = "https://api.github.com/repos/ANF-Studios/WinPath/releases";
```
Comment (ANF-Studios) [inline WinPath/src/Update.cs:18]: You're violating naming conventions.

File: WinPath/src/Update.cs [hunk 3]
```diff
@@ -85,9 +86,12 @@ internal void DownloadWinPath(in ReleaseInfo releaseInfo, Action finalJob = null
                 try
                 {
                     var application = Process.Start(process);
-                    application.WaitForExit();
-                    processExitCode = application.ExitCode;
-                    Console.WriteLine(application.ExitCode);
+                    if (application != null)
```
[the remark this answers is not in the document]
Comment (l-7777777) [reply WinPath/src/Update.cs:89]: in case `application` is null

**Comment by l-7777777** [comment]
wait how do I add another commit to this lmao

File: WinPath/src/Update.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (ANF-Studios) [reply WinPath/src/Update.cs:89]: I can tell that, I mean tell me a possibility of when/where it's null.

File: WinPath/src/Program.cs [hunk 4]
```diff
@@ -91,22 +91,23 @@ static void HandleArgument(HandleEventType eventType, AddOptions options = null)
...
-                            "There seems to be an error, we could not verify if that value is actually added to the Path or not, it's nothing to worry about though!"
+                    if (options != null)
+                    {
+                        userPath.AddToPath(
+                            options.Value,
+                            options.BackupPathVariable,
+                            DateTime.Now.ToFileTime().ToString()
                         );
+                        Console.WriteLine(
+                            @";C:\Users\lysan\AppData\Local\Microsoft\WindowsApps;C:\Users\lysan\AppData\Local\Programs\Microsoft VS Code\bin;C:\Program Files\heroku\bin;C:\Users\lysan\AppData\Roaming\npm;C:\Users\lysan\.dotnet\tools%PROGRAMFILES%\WinPath\;%PROGRAMFILES(X86)%\WinPath\;"
```
Comment (ANF-Studios) [inline WinPath/src/Program.cs:102]: ?

File: WinPath/src/Program.cs [hunk 5]
```diff
@@ -91,22 +91,23 @@ static void HandleArgument(HandleEventType eventType, AddOptions options = null)
...
-                            "Path",
-                            EnvironmentVariableTarget.User)
-                        .EndsWith(
-                            $"{options.Value};"
-                        )
-                    ) Console.WriteLine($"Successfully added `{options.Value}` to the Path!");
-                    else
-                        Console.WriteLine(
-                            "There seems to be an error, we could not verify if that value is actually added to the Path or not, it's nothing to worry about though!"
+                    if (options != null)
```
Comment (ANF-Studios) [inline WinPath/src/Program.cs:94]: This is not needed.

**Review by ANF-Studios — CHANGES_REQUESTED** [review]
A lot of stuff is wrong.
* Coding and formatting conventions are broken, mostly of the code base.
* You didn't have to even modify the gitignore.
* You committed your idea folder after.. ignoring it..?
* You changed the functionality of already working code that had no flaws.
* You have.. debug code? Of your own pc/config?
* Usings are not in order.
* You are violating C#'s naming conventions.
and more changes that are minor.

I do not approve, revert those changes back please.

File: WinPath/src/Update.cs [hunk 6]
```diff
@@ -126,52 +144,51 @@ internal void DownloadWinPath(in ReleaseInfo releaseInfo, Action finalJob = null
 
         internal List<Release> GetReleases()
         {
-            string response = String.Empty;
-            List<Release> releases;
             try
             {
+                string response;
```
[the remark this answers is not in the document]
Comment (lylythechosenone) [reply WinPath/src/Update.cs:149]: No, I removed that on purpose. It was unnecessary.

**Comment by ANF-Studios** [comment]
You were given a some time to implement that however you never did so I'm going to have to close this issue. If you want to discuss this, please reopen this pull request.

**Pull request closed without merging**